fedimint-server/src/config/api.rs

```diff
@@ -563,15 +597,17 @@ impl HasApiContext<ConfigGenApi> for ConfigGenApi {
             ConfigApiState::VerifyConfigParams(api_auth, _) => Some(api_auth) == auth,
             ConfigApiState::RunningDkg(api_auth) => Some(api_auth) == auth,
             ConfigApiState::FailedDkg(api_auth) => Some(api_auth) == auth,
-            ConfigApiState::VerifyConsensusConfig(api_auth, _) => Some(api_auth) == auth,
-            ConfigApiState::RunningConsensus(api_auth) => Some(api_auth) == auth,
+            ConfigApiState::VerifyConsensusConfig(cfg) => Some(&cfg.private.api_auth) == auth,
+            ConfigApiState::RunningConsensus(cfg) => Some(&cfg.private.api_auth) == auth,
         };
 
         (self, ApiEndpointContext::new(has_auth, dbtx, id))
     }
 }
 
-/// Starts the configuration server
+/// Starts the config gen server
```

Comment:
Nit, style: Newline between the "one line header" and rest of the docstring recommended.